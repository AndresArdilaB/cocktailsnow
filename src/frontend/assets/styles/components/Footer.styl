.footer
  align-items: center
  background-color: #000
  color: #FFF
  display: flex
  height: 100px
  width: 100%
  justify-content: center
  bottom: 0px
