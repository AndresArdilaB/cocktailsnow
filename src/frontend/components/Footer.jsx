import React from 'react';
import '../assets/styles/components/Footer.styl';

const Footer = () => (
  <footer className='footer'>
    From Andres Ardila for Platzi - 2020
  </footer>
);

export default Footer;
