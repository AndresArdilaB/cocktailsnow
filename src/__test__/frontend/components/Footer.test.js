import React from 'react';
import { mount } from 'enzyme';
import { create } from 'react-test-renderer';
import Footer from '../../../frontend/components/Footer';

describe('<Footer />', () => {
  const footer = mount(<Footer />);

  test('Footer component render', () => {
    expect(footer.length).toEqual(1);
  });

  test('Footer correct text', () => {
    expect(footer.find('.footer').text()).toEqual('From Andres Ardila for Platzi - 2020');
  });
});

describe('Footer Snapshot', () => {

  test('Testing the footer UI component', () => {
    const footer = create(<Footer />);
    expect(footer.toJSON()).toMatchSnapshot();
  });
});
