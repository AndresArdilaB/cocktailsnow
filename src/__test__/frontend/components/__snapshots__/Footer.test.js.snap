// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Snapshot Testing the footer UI component 1`] = `
<footer
  className="footer"
>
  From Andres Ardila for Platzi - 2020
</footer>
`;
